From: =?utf-8?q?Mateusz_=C5=81ukasik?= <mati75@linuxmint.pl>
Date: Mon, 22 Oct 2018 10:28:07 +0200
Subject: Add support for archive manager for MATE

Bug-Debian: https://bugs.debian.org/783611
---
 scripts/Makefile.am | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index e320358..6d0dc4f 100644
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -15,6 +15,11 @@ install-exec-hook:
 	-( cd $(DESTDIR)$(wrapperdir) ; \
 		test -f org.gnome.FileRoller.tap \
 		|| ln -sf file-roller.tap org.gnome.FileRoller.tap )
+# Install symlink to 'mate-engrampa.tap'
+	$(mkinstalldirs) $(DESTDIR)$(wrapperdir)
+	-( cd $(DESTDIR)$(wrapperdir) ; \
+		test -f mate-engrampa.tap \
+		|| ln -sf engrampa.tap mate-engrampa.tap )
 
 EXTRA_DIST =								\
 	$(wrapper_SCRIPTS)						\
