--- a/Makefile.in.in
+++ b/Makefile.in.in
@@ -74,8 +74,11 @@
 
 	install -d $(DESTDIR)/$(MANDIR)/man1
 	install xfce4-panel-profiles.1 $(DESTDIR)/$(MANDIR)/man1
-	
-	install -d $(DESTDIR)/$(PREFIX)/share/icons/hicolor/{128x128,48x48,16x16,scalable}/apps
+
+	install -d $(DESTDIR)/$(PREFIX)/share/icons/hicolor/128x128/apps
+	install -d $(DESTDIR)/$(PREFIX)/share/icons/hicolor/48x48/apps
+	install -d $(DESTDIR)/$(PREFIX)/share/icons/hicolor/16x16/apps
+	install -d $(DESTDIR)/$(PREFIX)/share/icons/hicolor/scalable/apps
 	install data/icons/128x128/org.xfce.PanelProfiles.png $(DESTDIR)/$(PREFIX)/share/icons/hicolor/128x128/apps
 	install data/icons/48x48/org.xfce.PanelProfiles.png $(DESTDIR)/$(PREFIX)/share/icons/hicolor/48x48/apps
 	install data/icons/16x16/org.xfce.PanelProfiles.png $(DESTDIR)/$(PREFIX)/share/icons/hicolor/16x16/apps
@@ -87,7 +90,9 @@
 	rm -rf $(DESTDIR)/$(DOCDIR)
 	# FIXME: Uninstall locales
 	rm -f $(DESTDIR)/$(PREFIX)/bin/$(APPNAME)
-	rm -rf $(DESTDIR)/$(PREFIX)/share/icons/hicolor/{128x128,48x48,16x16}/apps/org.xfce.PanelProfiles.png
+	rm -rf $(DESTDIR)/$(PREFIX)/share/icons/hicolor/128x128/apps/org.xfce.PanelProfiles.png
+	rm -rf $(DESTDIR)/$(PREFIX)/share/icons/hicolor/48x48/apps/org.xfce.PanelProfiles.png
+	rm -rf $(DESTDIR)/$(PREFIX)/share/icons/hicolor/16x16/apps/org.xfce.PanelProfiles.png
 	rm -rf $(DESTDIR)/$(PREFIX)/share/icons/hicolor/scalable/apps/org.xfce.PanelProfiles.svg
 
 distcheck: all
@@ -107,7 +112,9 @@
 	rm -f bin/$(APPNAME)
 	rm -f data/metainfo/org.xfce.PanelProfiles.appdata.xml
 	rm -f data/layouts/*.tar.bz2
-	rm -f data/icons/{128x128,48x48,16x16}/org.xfce.panelprofiles.png
+	rm -f data/icons/128x128/org.xfce.panelprofiles.png
+	rm -f data/icons/48x48/org.xfce.panelprofiles.png
+	rm -f data/icons/16x16/org.xfce.panelprofiles.png
 	rm -f data/icons/scalable/org.xfce.panelprofiles.svg
 	rm -f org.xfce.PanelProfiles.desktop
 	rm -f Makefile.in
